pkg/plugins/pfs/corelist/corelist_load_gen.go

```diff
@@ -0,0 +1,46 @@
+// THIS FILE IS GENERATED. EDITING IS FUTILE.
+//
+// Generated by:
+//     public/app/plugins/gen.go
+// Using jennies:
+//     PluginTreeListJenny
+//
+// Run 'make gen-cue' from repository root to regenerate.
+
+package corelist
+
+import (
+	"fmt"
+	"io/fs"
+
+	"github.com/grafana/grafana"
+	"github.com/grafana/grafana/pkg/plugins/pfs"
+	"github.com/grafana/thema"
+)
+
+func makeTreeOrPanic(path string, pkgname string, rt *thema.Runtime) *pfs.Tree {
+	sub, err := fs.Sub(grafana.CueSchemaFS, path)
+	if err != nil {
+		panic("could not create fs sub to " + path)
+	}
+	tree, err := pfs.ParsePluginFS(sub, rt)
+	if err != nil {
+		panic(fmt.Sprintf("error parsing plugin metadata for %s: %s", pkgname, err))
+	}
+	return tree
+}
+
+func coreTreeList(rt *thema.Runtime) pfs.TreeList {
+	return pfs.TreeList{
```

Comment:
i have no strong stance on whether trimming this list down is the right move or not. Rather, just a note:

My original thought in having all plugins in this generated list, regardless of whether they contain a models.cue, was that it might make bringing the pfs approach into the runtime plugin manager easier - this file was always guaranteed to have a statically enumerated list of all the core plugins. (modulo the ignored list, i guess?)

Reply:
Since this is not used by anything at the moment shouldn't be any problem to keep it as is and get this merged, or am I missing something dependent on this?

Reply:
https://github.com/grafana/schematization-and-as-code-project/issues/24 needs to be able to enumerate a list of plugins, including those that don't have composable kind implementations. @joanlopez discussed that one today, i believe he's tackling it.

grok also depends on it, though it doesn't really matter there

Reply:
Ok ok, I can refactor this one to keep the same behavior as prior to this PR.